tests/Microsoft.Bot.Builder.AI.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -962,7 +947,19 @@ public async Task Telemetry_ConvertParms()
             Assert.AreEqual(((Dictionary<string, double>)telemetryClient.Invocations[0].Arguments[2])["luis"], 1.0001);
         }
 
-
+        private static TurnContext GetContext(string utterance)
+        {
+            var b = new TestAdapter();
+            var a = new Activity
```

Comment:
Also here, can we rename it to `activity`??

Reply:
Yes, I will change both variable's name.

Reply:
Done